Trace-level retention filters evaluation edge case (#31643)
DataDog/documentation: content/en/monitors/types/apm.md

{{< img src="monitors/monitor_types/apm/define-the-search-query.png" alt="Define the search query" style="width:80%;" >}}

> **Note:** Analytics monitors only evaluate spans retained by [custom retention filters][6] (not the intelligent retention filter). Additionally, spans indirectly ndexed by [trace-level retention filters][7] (i.e. spans belonging to traces that contain spans matching the query, but not directly matching the query) are not evaluated by trace analytics monitors.

Reviewer: ```suggestion
**Note:** Analytics monitors only evaluate spans retained by [custom retention filters][6] (not the intelligent retention filter). Additionally, spans indirectly indexed by [trace-level retention filters][7] (that is, spans that don't match the query directly but belong to traces that do) are not evaluated by trace analytics monitors.
```
